### ISSUE TITLE
Context menu function issues.

### ISSUE THREAD

**Issue by bluej130** [careteditor/caret#111]
Any function in the context menu fails (excluding the cut, copy and paste) -- even after a clean install:

- Spell Correction
- Convert to OL
- Convert to Task List

Error code stuff:
```
{
    "message": "Cannot read property 'type' of undefined",
    "version": "1.6.0",
    "platform": "win32",
    "os": "10.0.14393",
    "path": "C:\\Users\\jdunham\\AppData\\Local\\Caret\\app-1.6.0\\resources\\app.asar\\web\\Editor\\Editor.Menu.js",
    "code": "\"search-google\"==t.type",
    "line": 5,
    "column": 2912,
    "stack": [
        "at List._menuSelect ((C:\\Users\\jdunham\\AppData\\Local\\Caret\\app-1.6.0\\resources\\app.asar\\web\\Editor\\Editor.Menu.js:5:2912)",
        "at Object.trigger ((C:\\Users\\jdunham\\AppData\\Local\\Caret\\app-1.6.0\\resources\\app.asar\\web\\Events.js:5:725)",
        "at Object.callback ((C:\\Users\\jdunham\\AppData\\Local\\Caret\\app-1.6.0\\resources\\app.asar\\packages\\Editor\\EditorDesktop.js:5:647)",
        "at Object.exec ((C:\\Users\\jdunham\\AppData\\Local\\Caret\\app-1.6.0\\resources\\app.asar\\web\\Extension.Command.js:5:997)",
        "at Object._onCommand ((C:\\Users\\jdunham\\AppData\\Local\\Caret\\app-1.6.0\\resources\\app.asar\\CaretDesktop.js:5:10160)",
        "at emitThree ((events.js:116:13)",
        "at EventEmitter.emit ((events.js:194:7)"
    ]
}
```

**Comment by astoilkov** [comment]
We are aware of the issue. You could try opening the menu using Ctrl+Space for now. We will fix it as soon as possible.

Thanks for reporting and sorry for the inconvenience.

**Comment by bluej130** [comment]
Oh man I didn't know `Ctrl`+`Space` was an option. Would totally rather use this option than right-clicking. It's not listed anywhere on your website or context menu. I even tried looking in [caret.io/releases](https://caret.io/releases) to see if I could find that feature. 

_(Originally discovered this was an option in your video on the homescreen.)_

Would be helpful if you had a simple list of commands in a help section somewhere.

**Comment by astoilkov** [comment]
We worked on such list today. It is still work in progress but we published it so it could benefit our users right away. You could check it out at https://caret.io/features.

**Comment by astoilkov** [comment]
This is now fixed in the latest 1.6.1 Caret release.

Thanks for reporting.
